tests/python/rest_api/test_tasks.py

```diff
@@ -2668,6 +2668,27 @@ def test_import_annotations_after_deleting_related_cloud_storage(
         task.import_annotations(self.import_format, file_path)
         self._check_annotations(task_id)
 
+    def test_check_import_error_on_wrong_file_structure(self, tasks_with_shapes):
```

Comment:
LGTM, but as this PR affects different formats, it makes sense to run this test for other formats as well. Consider adding `@pytest.mark.parametrize(format_name, ["...", ...])` with the list of formats.